src/valora-dapp-list.json

```diff
@@ -67,271 +67,347 @@
       "categoryId": "exchanges",
       "categories": ["exchanges"],
-      "url": "https://mento.finance/"
+      "url": "https://mento.finance/",
+      "listOnAndroid": true,
+      "listOnIos": true
     },
     {
       "name": "Flywallet",
       "id": "flywallet",
       "categoryId": "spend",
       "categories": ["spend", "social-impact"],
-      "url": "https://app.flywallet.io"
+      "url": "https://app.flywallet.io",
+      "listOnAndroid": true,
+      "listOnIos": true
     },
     {
       "name": "ImmortalDAO Finance",
       "id": "immortaldao-finance",
       "categoryId": "finance-tools",
       "categories": ["finance-tools", "earn"],
-      "url": "https://www.immortaldao.finance"
+      "url": "https://www.immortaldao.finance",
+      "listOnAndroid": true,
+      "listOnIos": true
     },
     {
       "name": "Alphafortress (Bitssa)",
       "id": "alphafortress",
       "categoryId": "exchanges",
       "categories": ["exchanges", "finance-tools"],
-      "url": "https://swap.bitssa.com"
+      "url": "https://swap.bitssa.com",
+      "listOnAndroid": true,
+      "listOnIos": true
     },
     {
       "name": "Trelis Pay",
       "id": "trelispay",
       "categoryId": "finance-tools",
       "categories": ["finance-tools"],
-      "url": "https://celo.trelis.com"
+      "url": "https://celo.trelis.com",
+      "listOnAndroid": true,
+      "listOnIos": true
     },
     {
       "name": "ChiSpend - Super App",
       "id": "chimoney-chispend",
       "categoryId": "spend",
       "categories": ["spend"],
-      "url": "https://chispend.com/?cSContext=valora"
+      "url": "https://chispend.com/?cSContext=valora",
+      "listOnAndroid": true,
+      "listOnIos": true
     },
     {
       "name": "HaloFi",
       "id": "halofi",
       "categoryId": "lend-borrow-earn",
       "categories": ["earn"],
-      "url": "https://app.halofi.me/#/?network=celo&utm_source=valora"
+      "url": "https://app.halofi.me/#/?network=celo&utm_source=valora",
+      "listOnAndroid": true,
+      "listOnIos": true
     },
     {
       "name": "Revo",
       "id": "revo",
       "categoryId": "lend-borrow-earn",
       "categories": ["earn"],
-      "url": "https://revo.market"
+      "url": "https://revo.market",
+      "listOnAndroid": true,
+      "listOnIos": true
     },
     {
       "name": "NFT Viewer",
       "id": "nftviewer",
       "categoryId": "nfts",
       "categories": ["nfts"],
-      "url": "https://nfts.valoraapp.com/nfts?address={{address}}&hide-header=true"
+      "url": "https://nfts.valoraapp.com/nfts?address={{address}}&hide-header=true",
+      "listOnAndroid": true,
+      "listOnIos": true
     },
     {
       "name": "Cyberbox",
       "id": "cyberbox",
       "categoryId": "nfts",
       "categories": ["nfts", "social-impact"],
       "url": "https://cyberbox.art/",
-      "canPurchaseNfts": true
+      "canPurchaseNfts": true,
+      "listOnAndroid": true,
+      "listOnIos": false
```

Comment:
Should we remove the canPurchasrNfts in a follow-up PR?

Reply:
Yes!